pkg/infra/appcontext/user.go

```diff
@@ -50,13 +51,21 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 			case k8suser.APIServerUser:
 				fallthrough
 			case k8suser.SystemPrivilegedGroup:
+				orgId := int64(1)
 				return &user.SignedInUser{
 					UserID:         1,
-					OrgID:          1,
+					OrgID:          orgId,
 					Name:           k8sUserInfo.GetName(),
 					Login:          k8sUserInfo.GetName(),
 					OrgRole:        roletype.RoleAdmin,
 					IsGrafanaAdmin: true,
+					Permissions: map[int64]map[string][]string{
+						orgId: {
+							"*":                            {"*"},
```

Comment:
the wildcard does not work for folders because it uses the permissions exactly -- while other resources (dashboards) process wildcards.  I'll leave it and then add in explicit permissions where necessary